### PR TITLE
adds notifications & switches tests to testnet

### PR DESCRIPTION
**Pull request by kenodressel**

### CODE REVIEW

File: logic/cacheLogic.js [hunk 1]
```diff
@@ -42,8 +42,8 @@ module.exports = class CacheLogic {
       await CacheLogic.getTipsAndVerifyLocalInfo();
       await CacheLogic.fetchChainNames();
       await CacheLogic.fetchPrice();
-      await aeternity.getOracleState();
       await CacheLogic.findContractEvents();
+      await aeternity.getOracleState();
```
Comment (thepiwo) [inline logic/cacheLogic.js:46]: why change the order here?

File: logic/cacheLogic.js [hunk 2]
```diff
@@ -52,7 +52,7 @@ module.exports = class CacheLogic {
   static async findContractEvents() {
     const fetchContractEvents = async () => {
       const contractTransactions = await aeternity.middlewareContractTransactions();
-      return contractTransactions.map(tx => tx.hash).asyncMap(aeternity.transactionEvents);
+      return contractTransactions.map(tx => tx.hash).asyncMap(hash => aeternity.transactionEvents(hash));
```
Comment (thepiwo) [inline logic/cacheLogic.js:55]: can this be reverted to original state?

File: logic/tipTracingLogic.js [hunk 3]
```diff
@@ -41,7 +41,7 @@ module.exports = class TipTracing {
 
     const contractTransactions = await aeternity.middlewareContractTransactions();
     const events = await contractTransactions.map(tx => tx.hash)
-      .asyncMap(aeternity.transactionEvents)
+      .asyncMap(hash => aeternity.transactionEvents(hash))
```
Comment (thepiwo) [inline logic/tipTracingLogic.js:44]: can this be reverted to original state?

File: logic/cacheLogic.js (hunk 2 above)
Comment (kenodressel) [reply logic/cacheLogic.js:55]: The original state is broken, so we can either
a) merge your PR soon
b) leave as is

File: logic/cacheLogic.js (hunk 1 above)
Comment (kenodressel) [reply logic/cacheLogic.js:46]: This was a test before I found the fix below, I will revert this.

**Pull request merged**